### PR TITLE
add a blog: ORAS 0.14 and Future: Empower Container Secure Supply Chain

### PR DESCRIPTION
**Pull request by FeynmanZhou**
Signed-off-by: Feynman Zhou <feynmanzhou@microsoft.com>

### CODE REVIEW

**Comment by FeynmanZhou** [comment]
@sajayantony Could you pls help to review this blog post? Steve and Lachie reviewed it for the first round last week.

File: docs/blog/oras-0.14-and-future.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# ORAS 0.14 and Future: Empower Container Secure Supply Chain
+
+_[Feynman Zhou](https://twitter.com/FeynmanZhou), CNCF Ambassador, ACR Product Manager_
+
+The [OCI Registry As Storage (ORAS)](https://oras.land/) project maintainers announced v0.14 release for the ORAS CLI recently. ORAS v0.14 introduces four new top-level commands and new options to manage supply chain artifacts across different container registries and multi-cloud environments. 
+
+## What's new in ORAS 0.14
+
+<figure>
+  <img src="/blog/oras-0.14-and-future/carbon.svg" />
```
Comment (SteveLasker) [inline docs/blog/oras-0.14-and-future.md:10]: The image doesn't seem to show in preview mode
I've also found .svg files dont' render consistently across devices. Which is a shame as the vector format would represent the best clarity. So, I've had to resort to png, with a white background to support dark mode views.

File: docs/blog/oras-0.14-and-future.md [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+</figure>
+
+
+Please see the [Release Notes](https://github.com/oras-project/oras/releases/tag/v0.14.0) for details.
+
+Prior to ORAS CLI v0.14 release, the ORAS Go library, also released v2.0.0-rc.2 to support [artifacts-spec v1.0.0-rc.2](https://github.com/oras-project/artifacts-spec/releases/tag/v1.0.0-rc.2) and provides new functions to enable developers to build your own OCI client tool.
+
+As cloud native development continues to grow, we have seen increased community interest in evolving registries to natively store, pull, copy, and discover a graph of supply chain artifacts. Artifact references are important for many use cases such as adding Software Bill of Materials (SBoM), security scan results, and container image signing. 
+
+This blog will demonstrate how to use ORAS CLI v0.14 to copy an image from a validated artifact registry to a personal conintainer registry, then attach a SBOM to it and discover the reference in a tree graph.
```
Comment (SteveLasker) [inline docs/blog/oras-0.14-and-future.md:20]: should we change "validated artifact registry" to "public registry"? I'm not sure what "validated" implies. Similiarly, personal, might be changed to private registry. You've also used "artifact registry" for the source, and container registry for the private version:

How about: 

> This blog will demonstrate how to use ORAS CLI v0.14 to copy an image from a public registry to a personal registry, then attach a SBOM to it and discover the reference in a tree graph.

File: docs/blog/oras-0.14-and-future.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+Please see the [Release Notes](https://github.com/oras-project/oras/releases/tag/v0.14.0) for details.
+
+Prior to ORAS CLI v0.14 release, the ORAS Go library, also released v2.0.0-rc.2 to support [artifacts-spec v1.0.0-rc.2](https://github.com/oras-project/artifacts-spec/releases/tag/v1.0.0-rc.2) and provides new functions to enable developers to build your own OCI client tool.
+
+As cloud native development continues to grow, we have seen increased community interest in evolving registries to natively store, pull, copy, and discover a graph of supply chain artifacts. Artifact references are important for many use cases such as adding Software Bill of Materials (SBoM), security scan results, and container image signing. 
+
+This blog will demonstrate how to use ORAS CLI v0.14 to copy an image from a validated artifact registry to a personal conintainer registry, then attach a SBOM to it and discover the reference in a tree graph. 
+
+![ORAS workflow](oras-0.14-and-future/Screenshot_1.png)
```
Comment (SteveLasker) [inline docs/blog/oras-0.14-and-future.md:22]: MCR is now MAR :)
Suggest changing the icon for SBOM from a docker whale to something more generic. If you're using SPDX, https://iconscout.com/icon/spdx-3629074

File: docs/blog/oras-0.14-and-future.md [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+tar -zxf oras_0.14.1_*.tar.gz -C oras-install/
+mv oras-install/oras /usr/local/bin/
+rm -rf oras_0.14.1_*.tar.gz oras-install/
+```
+
+> Note: You can also refer to the [installation guide](https://oras.land/cli/) for other different platforms.
+
+## Copy an image from Registry A to Registry B
+
+In this demo, assume all images are validated in MAR, so I will use ORAS to copy the container image from MAR to my personal repository of ACR. You can use your prefered container registry with ORAS.
```
Comment (SteveLasker) [inline docs/blog/oras-0.14-and-future.md:42]: Not sure what all images are validated in MAR implies.  
Suggest:

> In this demo we'll use ORAS to copy the container image from the public MAR registry to my private ACR registry. You can use your preferred container registry with ORAS.

nit, on preferred spelling

File: docs/blog/oras-0.14-and-future.md [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+
+## Using SBOM Tool to generate a SBOM
+
+An SBOM creates a machine-readable inventory of the software components that make up a given software product. Generating SBOM is a first step in Supply Chain Security. 
+
+You can use [Docker SBOM](https://docs.docker.com/engine/sbom/) or [SBOM Tool](https://github.com/microsoft/sbom-tool) to generate a SBOM for the target image.
+
+SBOM Tool can be used to create SPDX 2.2 compatible SBOMs for any variety of artifacts. In this demo, we use SBOM Tool to create SPDX 2.2 compatible SBOM for the sample Spark image.
+
+Install SBOM Tool on a Linux machine:
```
Comment (SteveLasker) [inline docs/blog/oras-0.14-and-future.md:56]: Suggest not saying "linux machine", as WSL is a linux environment, on a windows machine, and mac users don't necessarily consider them "linux machines".

> Install the SBOM Tool within a Linux environment:

I'd also suggest using some line wraps to make it easier to see:
```
sbom-tool generate -di feynmanacr.azurecr.io/mmlspark/spark2.4:1.0.0 \
  -b ./foo \
  -pn bar \
  -pv 0.1 \
  -bc ./foo \
  -ps MyCompany \
  -nsb http://mycompany.com

**Review by SteveLasker — COMMENTED** [review]
A few more nits:

File: docs/blog/oras-0.14-and-future.md [hunk 6]
```diff
@@ -0,0 +1,107 @@
...
+
+Artifact Type   Digest
+example/sbom    sha256:7592c8026675e463e7ced9b7ed369c2962b354a69b842423e8ctestdigest
+```
+
+## ORAS Present and Future
+
+ORAS has been integrated and adopted by some industry-leading ISVs and projects, such as [soci-snapshotter](https://github.com/awslabs/soci-snapshotter) by AWS, [KubeApps](https://github.com/vmware-tanzu/kubeapps) by VMware Tanzu, [UOR Framework](https://universalreference.io/) by Red Hat etc. 
+
+TBD: Introduce ORAS 0.15 and future milestones.
```
Comment (FeynmanZhou) [inline docs/blog/oras-0.14-and-future.md:100]: I will add this paragraph until https://github.com/oras-project/community/pull/21 is merged.

File: docs/blog/oras-0.14-and-future.md [hunk 7]
```diff
@@ -0,0 +1,107 @@
...
+<figure>
+  <img src="/blog/oras-0.14-and-future/carbon.svg" />
+</figure>
+
+
+Please see the [Release Notes](https://github.com/oras-project/oras/releases/tag/v0.14.0) for details.
+
+Prior to ORAS CLI v0.14 release, the ORAS Go library, also released v2.0.0-rc.2 to support [artifacts-spec v1.0.0-rc.2](https://github.com/oras-project/artifacts-spec/releases/tag/v1.0.0-rc.2) and provides new functions to enable developers to build your own OCI client tool.
+
+As cloud native development continues to grow, we have seen increased community interest in evolving registries to natively store, pull, copy, and discover a graph of supply chain artifacts. Artifact references are important for many use cases such as adding Software Bill of Materials (SBoM), security scan results, and container image signing.
```
Comment (sajayantony) [inline docs/blog/oras-0.14-and-future.md:18]:
```suggestion
As cloud native development continues to grow, we have seen increased community interest in evolving registries to natively store, pull, copy, and discover a graph of supply chain artifacts. Artifact references are important for many use cases such as adding Software Bill of Materials (SBoM), security scan results, and container image signatures. 
```

**Review by sajayantony — APPROVED** [review]

File: docs/blog/oras-0.14-and-future.md (hunk 2 above)
Comment (FeynmanZhou) [reply docs/blog/oras-0.14-and-future.md:20]: Yes. I was actually supposing all images in MAR are validated by Microsoft. That's the reason why we choose to copy the images from MAR as a trusted source.

How about:

> This blog will demonstrate how to use ORAS CLI v0.14 to copy an image from a public registry validated by Microsoft to a private registry, then attach an SBoM to it and discover the reference in a tree graph.

File: docs/blog/oras-0.14-and-future.md (hunk 1 above)
Comment (FeynmanZhou) [reply docs/blog/oras-0.14-and-future.md:10]: Let me convert it into png

File: docs/blog/oras-0.14-and-future.md (hunk 3 above)
Comment (FeynmanZhou) [reply docs/blog/oras-0.14-and-future.md]: Updated.

**Comment by FeynmanZhou** [comment]
Updated according to the comments above. @SteveLasker @sajayantony

**Pull request merged**